src/components/User-page/UserPage.js

```diff
@@ -0,0 +1,38 @@
+import React from "react";
+import {usersList} from "../../constants";
+import {UserCard} from "../user-card/UserCard";
+import {withRouter} from "react-router";
+
+
+class UserPageComponent extends React.Component{
+    constructor(props) {
+        super(props);
+    }
+
+
+    render() {
+        const { match: { params: { userId } }, history } = this.props;
+
+        const user = usersList.find(item => item.id === userId);
+
+        const toUsersList = () => {
```

Comment:
If you are using a class-based component it's better to define such functions as class' methods inside the body of the class. In this case, you would need to destruct this.props inside each of the methods.